aspnetcore/security/app-secrets.md

```diff
@@ -86,23 +76,17 @@ You can also use the Secret Manager tool to list, remove and clear app secrets.
 
 ## Accessing user secrets via configuration
 
-You access Secret Manager secrets through the configuration system. Add the `Microsoft.Extensions.Configuration.UserSecrets` as a dependency in your *project.json* file and run `dotnet restore`.
+You access Secret Manager secrets through the configuration system. Add the `Microsoft.Extensions.Configuration.UserSecrets` to the *.csproj* file and run `dotnet restore`.
```

Comment:
"to the project **file**" still assumes you're editing a file, which you might not be doing if you're using VS.  In the docs that we updated to remove project.json references in master we just changed it to "add (or install) the x package."